notebooks/lesson09-overfitting.ipynb

```diff
@@ -0,0 +1,2113 @@
+{
```

Comment:
<ul><li>i think you can add these steps to binder by adding a <code>postBuild</code>  file as done here: <a href="https://github.com/binder-examples/jupyter-extension/blob/master/postBuild" target="_blank">https://github.com/binder-examples/jupyter-extension/blob/master/postBuild</a></li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lewtun/dslectures/pull/32/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>It seems that <code>tqdm</code>  is working out of the box in binder. I just added it for the students that want to install everything locally.</p>